pages/products.html

```diff
@@ -4,10 +4,553 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://ampereui.netlify.app/ampereUI.css" />
     <link rel="stylesheet" href="../styles/products.css" />
     <title>Products</title>
   </head>
   <body>
-    <h1>Products</h1>
+    <div id="header">
+      <div class="navigation-container">
+        <div class="upper-container">
+          <div class="logo-and-menu">
+            <i class="drawer-menu-icon uil uil-bars"></i>
```

Comment:
It's good to wrap the main nav in a "header" tag and use semantic HTML tags like <nav>